### PR TITLE
python: Support binOp function calls.

### PR DESCRIPTION
**Pull request by tebeka**
Happend when user has an expression like (datetime.now() - start).total_second()

Fixes ENG-1213

### CODE REVIEW

File: runtimes/pythonrt/ak_runner/loader.py [hunk 1]
```diff
@@ -26,6 +42,12 @@ def name_of(node):
     if isinstance(node, ast.Name):
         return node.id
 
+    if isinstance(node, ast.BinOp):
+        left = name_of(node.left)
+        right = name_of(node.right)
+        op = bin_ops.get(type(node.op), "?")
```
Comment (itayd) [inline runtimes/pythonrt/ak_runner/loader.py:48]: shouldn't this be an error?

**Review by itayd — COMMENTED** [review]
can you explain the fix?

File: runtimes/pythonrt/ak_runner/loader.py (hunk 1 above)
Comment (tebeka) [reply runtimes/pythonrt/ak_runner/loader.py:48]: The code that caused it is `(datetime.now() - start).total_seconds()`, not a bug.

**Comment by tebeka** [comment]
> can you explain the fix?

Support code like: `(datetime.now() - start).total_seconds()`

**Review by daabr — APPROVED** [review]

**Comment by itayd** [comment]
> > can you explain the fix?
> 
> Support code like: `(datetime.now() - start).total_seconds()`

you're funny. i mean how does it do that? not clear to me.

**Pull request merged**